### ISSUE TITLE
[Accessibility] Search button on blog page does not have ARIA label

### ISSUE THREAD

**Issue by mikaylaorange** [blaccsmith/website#62]
### Describe the bug

Search button missing ARIA label

### To Reproduce

1. Go to the [blog site](https://blog.blacc.xyz)
2. Turn on screenreader
3. Navigate to search button
4. See that the screenreader only announces "button"

[Screen Recording](https://watch.screencastify.com/v/zKaLDWUCFJ4PvETRsJUC)

### Expected Behavior

Search button should have ARIA label like "Search" or "Search Blogs"

### Where is this happening

- [X] Desktop
- [ ] Mobile

### Addtional Context

Testing using VoiceOver on MacOS
